### PR TITLE
k8s for NATS

### PR DESCRIPTION
**Pull request by ajax-marchenko-s**

### CODE REVIEW

File: gateway/src/main/kotlin/ua/marchenko/gateway/common/nats/NatsConnectionConfiguration.kt [hunk 1]
```diff
@@ -2,11 +2,12 @@ package ua.marchenko.gateway.common.nats
 
 import io.nats.client.Connection
 import io.nats.client.Nats
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 
 @Configuration
-class NatsConnectionConfiguration {
+class NatsConnectionConfiguration(@Value("\${nats.uri}") val natsUri: String) {
```
Comment (ajax-tarnavskyi-m) [inline gateway/src/main/kotlin/ua/marchenko/gateway/common/nats/NatsConnectionConfiguration.kt:10]:
```suggestion
class NatsConnectionConfiguration(@Value("\${nats.uri}") private val natsUri: String) {
```
*And same for domain-service nats config

**Review by ajax-tarnavskyi-m — COMMENTED** [review]
Great Job!

**Review by ajax-samondros-b — APPROVED** [review]

File: k8s-configuration/gateway-deployment.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  template:
+    metadata:
+      name: gateway
+      labels:
+        app: gateway
+    spec:
+      containers:
+        - name: gateway
+          image: gateway:latest
+          imagePullPolicy: IfNotPresent
```
Comment (ajax-yurkovskyi-t) [inline k8s-configuration/gateway-deployment.yml:21]: specify version

File: k8s-configuration/artauction-deployment.yml [hunk 3]
```diff
@@ -67,5 +78,5 @@ spec:
     app: app
   ports:
     - protocol: TCP
-      port: 8080
-      targetPort: 8080
+      port: 8081
+      targetPort: 8081
```
Comment (ajax-yurkovskyi-t) [inline k8s-configuration/artauction-deployment.yml:82]: will this work with mongo-express?

Comment (ajax-marchenko-s) [reply k8s-configuration/artauction-deployment.yml:82]: remove mongo express cause we are using compass

**Pull request merged**